services/bettersocial/fcmToken/sendMultiDeviceTaggedNotification.js

```diff
@@ -29,9 +29,10 @@ const sendMultiDeviceTaggedNotification = async (
         data: {
           feed_id: activity_id,
           type: 'feed'
-        }
+        },
+        token: user?.token
       };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `sendToDevice` to `send` is a significant one. The `send` method requires the token to be part of the payload, while `sendToDevice` takes it as a separate argument. This change seems to be correctly implemented here.

However, there's a potential issue with error handling. If `user?.token` is `undefined`, the `send` method might throw an error. It would be better to check if the token exists before attempting to send the message.

```diff
        },
- 33:         token: user?.token
+ 33:         token: user?.token || ''
       };
      await messaging().send(payload);
```
This way, if `user?.token` is `undefined`, an empty string will be used instead, preventing any potential errors from being thrown by the `send` method.


<!-- This is an auto-generated comment by OSS CodeRabbit -->